app/Resources/views/field_of_study/show_all.html.twig

```diff
@@ -1,53 +1,74 @@
 {% extends 'adminBase.html.twig' %}
 
 {% block breadcrumb %}
-    <li class="breadcrumb-item active">
-        Linjer
-    </li>
+	<li class="breadcrumb-item">
+		{% if is_granted_team_leader() %}
+			<a href="{{ path('show_field_of_studies') }}">
+			{% else %}
+				<a href="{{ path('show_field_of_studies') }}">
+				{% endif %}
+				Linjer
+			</a>
+		</li>
+	</li>
```

Comment:
There seem to be two closing `li` tags, but only one opening?
The extra one should be removed.
![image](https://user-images.githubusercontent.com/46197518/202720463-291b4a0e-6438-48bd-a928-e20c85260928.png)